src/app/event-view-external/event-view-external.component.js

```diff
@@ -0,0 +1,15 @@
+(function(angular) {
+  'use strict';
+
+  angular.module('esn.calendar')
+  .component('calEventViewExternal', {
+    bindings: {
+      eventJcal: '<',
+      attendeeEmail: '<',
+      links: '<'
+    },
+    controller: /* @ngInject */ 'CalEventViewExternalController',
```

Comment:
THis ngInject is useless

Reply:
You must `require` the js file containing the controller on the top of the file

Reply:
The controller is required in the app/app.js file. So I think we don't need to require in the component again

Reply:
Also, I removed the `/* @ngInject */`